app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListFragment.kt

```diff
@@ -53,8 +53,15 @@ class TransactionListFragment : SafeOverviewBaseFragment<FragmentTransactionList
 
         adapter.addLoadStateListener { loadState ->
             if (lifecycle.currentState.isAtLeast(Lifecycle.State.STARTED)) {
-                binding.transactions.isVisible = loadState.refresh is LoadState.NotLoading
                 binding.progress.isVisible = loadState.refresh is LoadState.Loading
```

Comment:
Is this the full page loading spinner?

Reply:
yes

Reply:
And it shows whenever we load data?

Reply:
only if the list is empty. For lists with content we show only pull to refresh spinner